doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -1456,8 +1456,8 @@ case-insensitive.
    |             |               |            | Starting from RF 3.2.2, matching members is case-, space-      |        RED = 1                       |
    |             |               |            | and underscore-insensitive.                                    |        GREEN = 2                     |
    |             |               |            |                                                                |        DARK_GREEN = 3                |
-   |             |               |            |                                                                |                                      |
-   |             |               |            |                                                                | | `GREEN` (Color.GREEN)              |
+   |             |               |            | Starting from RF 4.0 `NONE` is not converted and will result   |                                      |
+   |             |               |            | failure if matching enum value is not found.                   | | `GREEN` (Color.GREEN)              |
```

Comment:
This explanation may not be that clear. Because this is a bug fix and because `NONE` as an enum member is _very_ rare, I'd say we don't need to have any explanation regarding to it.

A bit above this place there's documentation saying that `NONE`, case-insensitively, is converted to  Python `None`. That needs to be changed/removed and a note added that this behavior has changed in RF 4.0.